crates/driver/src/infra/api/quote/dto/order.rs

```diff
@@ -41,3 +45,9 @@ enum Kind {
     Sell,
     Buy,
 }
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("received an order with an exceeded deadline")]
```

Comment:
We refer to the "quote request" as the "quote order". It's neither here nor there and I wouldn't mind renaming `quote::Order` to `quote::Request` but I don't think that either name is much better. I personally avoid the word `Request` like the plague since it's so damn pervasive.